katas/content/nonlocal_games/examples/CHSHGameDemo.qs

```diff
@@ -0,0 +1,53 @@
+namespace Quantum.Kata.CHSHGame {
+    open Microsoft.Quantum.Random;
+    open Microsoft.Quantum.Math;
+    open Microsoft.Quantum.Convert;
+
+    function WinCondition (x : Bool, y : Bool, a : Bool, b : Bool) : Bool {
+        return (x and y) == (a != b);
+    }
+
+    function AliceClassical (x : Bool) : Bool {
+        return false;
+    }
+
+    function BobClassical (y : Bool) : Bool {
+        return false;
+    }
+
+    operation AliceQuantum (bit : Bool, qubit : Qubit) : Bool {
+        if bit {
+            return MResetX(qubit) == One;
+        }
+        return MResetZ(qubit) == One;
+    }
+
+    operation BobQuantum (bit : Bool, qubit : Qubit) : Bool {
+        let angle = 2.0 * PI() / 8.0;
+        Ry(not bit ? -angle | angle, qubit);
+        return M(qubit) == One;
+    }
+
+    @EntryPoint()
+    operation CHSH_GameDemo() : Unit {
+        use aliceQubit = Qubit();
+        use bobQubit = Qubit();
```

Comment:
Minor, but this can be shortened a bit if we allocate a tuple right away
```suggestion
        use (aliceQubit, bobQubit) = (Qubit(), Qubit());
```